WTWSMS/history/provinces/676 - Kakheti.txt

```diff
@@ -15,8 +15,12 @@ b_davidgareja = temple
 #b_gremi = castle
 
 # Misc
-culture = armenian
-religion = georgian_orthodox
+culture = juhuro
+religion = tzadikite
```

Comment:
[http://www.azerb.com/az-mountain-jews.html](This article) seems to indicate that the Juhurim first arrived in the game's location, in tiny numbers, during the 700s _BC_. However, they settled en masse in the high mountains of Caucasian Albania during 400-500 AD, which was this Mod's time period. They settled in remote, mountaianous areas where few others went. Even to this day, there are still Juhuro-majority and all-Juhuro cities in that area.

One statement that keeps coming back is that [Mountain Jews dominated Shamakh](https://www.revolvy.com/topic/History%20of%20the%20Jews%20in%20Azerbaijan&uid=null) until the 1920s AD. In more ancient times, they were far more numerous.

>"Historically, Jews in Azerbaijan have been represented by various subgroups, mainly Azerbaijani Jews, Ashkenazi Jews and Georgian Jews. Azerbaijan at one point was or still is home to smaller communities of Krymchaks, Kurdish Jews and Bukharian Jews, as well Gerim (converts) and non-Jewish Judaistic groups like Subbotniks. In those days, Jews used to live in and around the city of Shamakhi (mainly in the village of Mücü), but the community has been non-existent since the early 1920s."

I think I'm right about the Mountain Jews being a majority in a Caucasian Albanian province, but it may not be Kakheti. I think the most historically accurate representation would be to create and split off a smaller province in the area around Shamakh and have the Mountain Jews run that.

However, if that's too much work, we should simulate the Juhurim majority by keeping them a majority in Kakheti.

Here is an article [from the Jewish Encyclopedia](http://www.jewishencyclopedia.com/articles/4156-caucasus) concerning the Mountain Jews, and other Caucasian Jews.